Library/Homebrew/formulary.rb

```diff
@@ -276,6 +276,10 @@ def self.canonical_name(ref)
     ref.downcase
   end
 
+  def self.formula_ref?(ref)
+    !loader_for(ref).is_a? NullLoader
+  end
```

Comment:
Is this still needed? This looks like a leftover from the original implementation of this PR.